Log events to Stdout

jonnyarnold opened the pull request:
We will now log events to Stdout. This will be picked up by FluentD, according to @errm :dancer:

Commit:
Events have an ID

Commit:
Moved to NewEvent constructor

Commit:
UUID testing

Commit:
Replay partial commit e5d34d

Commit:
Using Event Logging instead of DynamoDB push

Commit:
[+EG] WIP

erdemgezer committed:
Test EventLog writes to STDOUT with correct JSON

erdemgezer committed:
Godeps

Commit:
Whoops

reevoo-samuel commented:
_If you're new, read the [guidelines](https://github.com/reevoo/guidelines/blob/master/pull_requests.md)._

Good code is **tested**, **easy to understand** and **able to cope when things go wrong**.

Good pull requests are **descriptive**, **small** and **short-lived**.

**For the Author**
- [x] Description explains what you have done and why.
- [x] Tests have been written (or you have explained why you don't need tests).
- [x] Documentation/comments have been added/updated.

**For the Reviewer**
- [x] Tests are green on CI (or pulled/tested on your machine).
- [x] Tests cover happy and unhappy paths in the code.
- [x] The code is easy to understand.

erdemgezer committed:
[+JA] New line after each event log and refactor EventLog tests
- Add new line after each stored event
- Refactor EventLog test
- Remove unused NewSilentServer constructor

erdemgezer committed:
Rename EventLog to EventLogger

erdemgezer committed:
Use EventLogger in the server after renaming

errm commented:
Looks good to me . . .
